### ISSUE TITLE
samtools sort error in clockwork:alignToRef

### ISSUE THREAD

**Issue by annacprice** [Pathogen-Genomics-Cymru/tb-pipeline#4]
Following error has been recorded:

```
[ERROR] failed to open file 'null'
[bam_mating_core] ERROR: Couldn't read header
samtools sort: failed to read header from "-"
[markdup] error reading header
```

**Comment by annacprice** [comment]
clockwork:alignToRef fails when the top species hit does not have a corresponding reference genome. This error is expected and recorded in the error log.

**Comment by annacprice** [comment]
Error is recorded correctly, but samples should not be passing to clockwork:alignToRef when the top species hit does not have a corresponding reference genome

**Comment by annacprice** [comment]
Closed by 12a9412bdb2bceaea1e2018d2c6dceb43ff259ed
